src/components/Remark.tsx

```diff
@@ -52,13 +58,14 @@ const Remark = ({ className, targetChain }: Props) => {
     setIsExecuting(true);
 
     try {
-      const keyring = new Keyring({ type: 'sr25519' });
-      const account = keyring.addFromUri('//Alice');
-
+      if (!currentAccount) {
+        return;
+      }
+      const account = currentAccount;
```

Comment:
```suggestion
      if (!account) {
        return;
      }
```

Reply:
This was addressed already in a new PR... but yes ... that does not makes sense.